src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Analyze/ConsoleAnalyzeCommand.cs

```diff
@@ -18,7 +18,7 @@ public ConsoleAnalyzeCommand()
                 Host.CreateDefaultBuilder()
                     .UseConsoleUpgradeAssistant<ConsoleAnalyze>(options, result)
                     .RunUpgradeAssistantAsync(token));
-
+                    
```

Comment:
nit: remove

There's probably a warning locally and will be an error on build for this. Make sure you clean up any warnings locally before pushing or the CI will fail.

Reply:
Resolved